wp-content/plugins/zippicks-smart-search/includes/class-search-engine.php

```
@@ -0,0 +1,796 @@
+        }
+    }
+    
+    /**
+     * Search by vibe
+     * 
+     * @param array $classification
+     * @param array $params
+     * @param array $context
+     * @return array
+     */
+    private function search_by_vibe($classification, $params, $context) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$context'. (Unused Code Rules)

(UnusedFormalParameter)